x-pack/libbeat/common/cloudfoundry/events.go

```diff
@@ -492,9 +492,9 @@ func envelopMap(evt Event) common.MapStr {
 func baseMap(evt Event) common.MapStr {
 	return common.MapStr{
 		"cloudfoundry": common.MapStr{
+			"type": evt.String,
```

Comment:
Should we call the function?
```suggestion
			"type": evt.String(),
```

Reply:
Definitely should call the function!! Fixed.